### PR TITLE
Added Principal Component Analysis

### PR DESCRIPTION
**Pull request by pps-19012**
Added PCA and implementation of PCA in Face Recognition

## Changes: 
1. PCA .ipynb file + dataset
2. Face Recognition folder - 1 pdf + 1 .ipynb + dataset

## Type of change

- [ ] Documentation update
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New layer/algorithm

### Implementation details

> * List all the packages and their versions used.
Mentioned in README.

## Tests

Run the i.pynb files. It has README instructions to run successfully.

## Checklist

- [ ] I have performed a self code_review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by HemanthSai7 — COMMENTED** [review]
@pps-19012 you previously made a PR on PCA. Why again?

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
       * [Logistic Regression](./traditional_ML_algorithms/LogisticRegression)
       * [KNN](./traditional_ML_algorithms/KNN)
       * [K-Means](./traditional_ML_algorithms/K-Means)
+      * [Principal Component Analysis](traditional_ML_algorithms/Principal%20Component%20Analysis)
```
Comment (mayureshagashe2105) [inline README.md:22]: Resolve the merge conflict first.

File: traditional_ML_algorithms/Principal Component Analysis/PCA_FaceRecognition/readme.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Implementation of Principal Component Analysis
+Face Recognition is an important implementation of PCA. It uses eigenvectors of the covariance vectors of the image matrix to detect and recognize faces.
```
Comment (mayureshagashe2105) [inline traditional_ML_algorithms/Principal Component Analysis/PCA_FaceRecognition/readme.md:2]: Resolve the merge conflict first!

File: traditional_ML_algorithms/Principal Component Analysis/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Principal Component Analysis
+
+### Packages used:
+
```
Comment (mayureshagashe2105) [inline traditional_ML_algorithms/Principal Component Analysis/README.md:4]: Resolve the merge conflict first!

**Review by mayureshagashe2105 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts and please explain why have you opened this PR again.

**Comment by pps-19012** [comment]
The previous PR had implementation of algorithm only on a specific topic (Face Recognition). Now, it also has general implementation.

**Comment by mayureshagashe2105** [comment]
Alright, sounds good! Please resolve the merge conflicts.

**Comment by pps-19012** [comment]
Done!

**Review by HemanthSai7 — APPROVED** [review]
LGTM🚀. Merging

**Comment by mayureshagashe2105** [comment]
LGTM!

**Review by mayureshagashe2105 — APPROVED** [review]

**Pull request merged**